src/components/testCaseList/TestCaseList.tsx

```diff
@@ -65,46 +65,51 @@ const TestCaseList = () => {
+          measureId
+        );
+        const executionResults: ExecutionResult[] =
+          await calculation.current.calculateTestCases(
+            measure,
+            testCases,
+            measureBundle
+          );
+
+        testCases.forEach((testCase) => {
+          const { populationResults } = executionResults.find(
+            (result) => result.patientId === testCase.id
+          )?.detailedResults?.[0]; // Since we have only 1 population group
 
-            const populationValues =
-              testCase?.groupPopulations?.[0]?.populationValues;
+          const populationValues =
+            testCase?.groupPopulations?.[0]?.populationValues;
 
-            // executionStatus is set to false if any of the populationResults (calculation result) doesn't match with populationValues (Given from testCase)
-            if (populationResults && populationValues) {
-              let executionStatus = true;
-              populationResults.forEach((populationResult) => {
-                if (executionStatus) {
-                  const groupPopulation = populationValues.find(
-                    (populationValue) =>
-                      getFhirMeasurePopulationCode(populationValue.name) ===
-                      populationResult.populationType.toString()
-                  );
+          // executionStatus is set to false if any of the populationResults (calculation result) doesn't match with populationValues (Given from testCase)
+          if (populationResults && populationValues) {
+            let executionStatus = true;
+            populationResults.forEach((populationResult) => {
+              if (executionStatus) {
+                const groupPopulation = populationValues.find(
+                  (populationValue) =>
+                    getFhirMeasurePopulationCode(populationValue.name) ===
+                    populationResult.populationType.toString()
+                );
 
-                  if (groupPopulation) {
-                    groupPopulation.actual = populationResult.result;
-                    executionStatus =
-                      groupPopulation.expected === populationResult.result;
-                  }
+                if (groupPopulation) {
+                  groupPopulation.actual = populationResult.result;
+                  executionStatus =
+                    groupPopulation.expected === populationResult.result;
                 }
-              });
-              testCase.executionStatus = executionStatus ? "pass" : "fail";
-            }
-          });
-          setTestCases([...testCases]);
-        })
-        .catch((error) => {
-          setError(error.message);
+              }
+            });
+            testCase.executionStatus = executionStatus ? "pass" : "fail";
+          }
         });
+        setTestCases([...testCases]);
+      } catch (error) {
+        console.error("An error occurred while executing test cases", error);
+        setError(error.message);
```

Comment:
Do we need to log these errors in UI?
I am not exactly sure about the best practices,  but it just logs on browser console, and I am wondering if they are of any use.

Reply:
I mean we never know when these errors could be helpful to the user. 
The basic use of logs is to help us debug an issue when something unexpected happens. but we are providing a message to the user via "SetError". So wouldn't that be self-sufficient to help us identify the issue during a HelpDesk Ticket?  

I am not questioning exactly, but I am thinking out loud.

Reply:
I don't know, I can't think of any use for these logs unless we use some kind of tool to capture them (Ex: Sentry).

Reply:
Usually the error message we're giving to the user is something generic like "An error occurred with test case execution." For example, if there was an error in execution that didn't stem from a backend service, we'd be absorbing the javascript/typescript error and wouldn't be able to troubleshoot outside of reproducing it locally.
I'll remove the error logging, but I feel it's more likely that we'll need to replicate issues locally for troubleshooting and debugging.

Reply:
console statements removed from create test case component